tests/functional/treeviewer/test_links_and_infoboxes.py

```diff
@@ -36,7 +36,7 @@ def test_MD_nolinks(self):
             #only get the ones with an id
             id = elem.get_attribute("id")
             if id:
-                yield self.MD_nolinks, id
```

Comment:
Just doing something simple for now, but could in principle do some fancier fixture stuff here for better reporting